lib/bap_image/sectionRef.mli

```diff
@@ -0,0 +1,38 @@
+(** Signle section in the list of sections int the object file *)
```

Comment:
Here you can use normal exceptions instead of `Or_error` monads, since this is external functions. Creating exceptions from C is pretty easy, but trying to create `Or_error` (especially the `Error`) part in C is very hard, first of all you need to export `Or_error` module to C. Either use exceptions or `option` type.